pkg/apiserver/openapi/openapi.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openapi
+
+import (
+	"bytes"
+	"github.com/emicklei/go-restful"
+	"strings"
+	"unicode"
+)
+
+// toValidOpId makes an string a valid op ID (e.g. removing punctuations and whitespaces and make it camel case)
+func toValidOpId(s string, capitalizeFirstLetter bool) string {
```

Comment:
if you need to make a change to fix the test, might as well change this to toValidOperationID or toValidOpID-- go style does ID instead of Id for whatever reason.